### ISSUE TITLE
Wrong place for generated types definition

### ISSUE THREAD

**Issue by Mikastark** [graphql-nexus/nexus-plugin-prisma#874]
Hello there :)

When I use `makeSchema`, the **nexusSchemaPrisma@0.20.0** plugin generate by default typescript definition (@types/typegen-nexus-plugin-prisma/index.d.ts) inside my working directory instead of node_modules. The default path is supposed to be **node_modules/@types/typegen-nexus-plugin-prisma/index.d.ts**

Here is my schema.ts file (it is pretty standard)
```ts
import { makeSchema } from '@nexus/schema'
import { nexusSchemaPrisma } from 'nexus-plugin-prisma/schema'
import * as types from './types'

export const schema = makeSchema({
  types,
  plugins: [
    nexusSchemaPrisma({
      experimentalCRUD: true,
      paginationStrategy: 'prisma',
    }),
  ],
  outputs: {
    schema: true,
  },
  typegenAutoConfig: {
    sources: [
      {
        source: '@prisma/client',
        alias: 'client',
      },
      {
        source: require.resolve('./context'),
        alias: 'Context',
      },
    ],
    contextType: 'Context.Context',
  },
})
```

**Comment by JustinVoitel** [comment]
there is a workaround as described in #873

**Comment by Mikastark** [comment]
The workaround doesn’t work.

**Comment by ziimakc** [comment]
same problem

**Comment by vimutti77** [comment]
```
nexusSchemaPrisma({
  experimentalCRUD: true,
  paginationStrategy: 'prisma',
  outputs: {
    typegen: path.join(process.cwd(), 'node_modules/@types/typegen-nexus-plugin-prisma/index.d.ts'),
  },
}),
```
This workaround is work for me

**Comment by Weakky** [comment]
Closed by #884
